addons/website/static/src/components/fullscreen_indication/fullscreen_indication.js

```diff
@@ -25,7 +25,7 @@ export class FullscreenIndication extends Component {
     }
 
     get fullScreenIndicationText() {
-        return markup(sprintf(escape(_t("Press %(key)s to exit full string")), "<span>esc</span>"));
+        return markup(sprintf(escape(_t("Press %(key)s to exit fullscreen")), "<span>esc</span>"));
```

Comment:
```suggestion
        return markup(sprintf(escape(_t("Press %(key)s to exit full screen")), "<span>esc</span>"));
```
As before the mistake made 3 days ago with https://github.com/odoo/odoo/commit/02461534d2b74410bcdfb5cccaac8157fe806127